backend/authentication-microservice/src/main/java/com/ss/utopia/auth/entity/UserRole.java

```diff
@@ -22,6 +25,24 @@ public UserRole(Long id, String name) {
     this.id = id;
     this.name = name;
   }
+  
+  public UserRole(String name) {
+	  String role = name.toUpperCase();
+	  switch(role) {
+	  	case "ADMIN": 
+	  		this.id = 1L;
+	  		break;
+	  	case "Customer":
+	  		this.id = 2L;
+	  		break;
+	  	case "AGENT":
+	  		this.id = 3L;
+	  		break;
+	  	default:
+	  		throw new HttpServerErrorException(HttpStatus.BAD_REQUEST, "Unauthorized Role");
```

Comment:
You don't wanna throw the error in the model

Reply:
I will have it throw the error upstream